squad/api/rest.py

```diff
@@ -430,6 +430,32 @@ class ProjectViewSet(viewsets.ModelViewSet):
     """
     List of projects. Includes public projects and projects that the current
     user has access to.
+
+    Additional actions:
+
+     * `api/projects/<id>/builds` GET
+
+        List of builds for the current project.
+
+     * `api/projects/<id>/suites` GET
+
+        List of test suite names available in this project
+
+     * `api/projects/<id>/tests` GET
+
+        List of test names available in this project
+
+     * `api/projects/<id>/test_results` GET
```

Comment:
This exposed a bug when no `test_name` is passed in the querystring: https://github.com/Linaro/squad/issues/729